components/form/FormItem.tsx

```diff
@@ -37,12 +37,16 @@ type ChildrenType<Values = any> = RenderChildren<Values> | React.ReactNode;
 interface MemoInputProps {
   value: any;
   update: any;
+  childProps: any;
   children: React.ReactNode;
 }
 
 const MemoInput = React.memo(
   ({ children }: MemoInputProps) => children as JSX.Element,
-  (prev, next) => prev.value === next.value && prev.update === next.update,
+  (prev, next) =>
+    prev.value === next.value &&
+    prev.update === next.update &&
+    JSON.stringify(prev.childProps) === JSON.stringify(next.childProps),
```

Comment:
Do not compare all the props. And this also make throw if:
```
const myProps = {};
myProps.aaa = myProps;

<Form.Item {...}>
  <Input {...myProps} />
</Form.Item>
```

Reply:
Thanks for the review. Can you explain what you want to accomplish with the line in generel?

We need to check the childProps as they might change as well. In this case we need to modify the element. So I should check individuell child props? Like aria labels?

Reply:
Seems just pass additional `aria-xxx` for compare is enough. Add more specific props to compare on MemoInput instead of fully props.